stepconfig/example_test.go

```diff
@@ -0,0 +1,72 @@
+package stepconfig_test
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/bitrise-tools/go-steputils/stepconfig"
+)
+
+type Configuration struct {
+
+	// Env vars specified in the struct tags are converted to the respective basic data types.
+	Name        string `env:"name"`
+	BuildNumber int    `env:"build_number"`
+	IsUpdate    bool   `env:"is_update"`
+
+	// List items have to be separated by pipe '|', like: "item1|item2"
+	Items []string `env:"items"`
+
+	// Secrets are not shown in the output.
+	Password stepconfig.Secret `env:"password"`
+
+	// If the env var is not set, the field will be set to the type's default value.
+	Empty string `env:"empty"`
+
+	// Env vars marked as 'required' have to be set.
+	Mandatory string `env:"mandatory,required"`
+
+	// File validation is checks if the file is exist in the specified path.
```

Comment:
File validation is checks if the file is exist in the specified path. -> File validation checks if the file exists in the specified path.